packages/frontend/src/pages/Playground.svelte

```diff
@@ -1,5 +1,151 @@
 <script lang="ts">
+import { conversations } from '../stores/conversations';
+import { studioClient } from '/@/utils/client';
+import {
+  isAssistantChat,
+  type ChatMessage,
+  type UserChat,
+  isPendingChat,
+  isUserChat,
+  type AssistantChat,
+} from '@shared/src/models/IPlaygroundMessage';
+import NavPage from '../lib/NavPage.svelte';
+import { playgrounds } from '../stores/playgrounds-v2';
+import { catalog } from '../stores/catalog';
+import Button from '../lib/button/Button.svelte';
+import { afterUpdate } from 'svelte';
+
 export let playgroundId: string;
+let prompt: string;
```

Comment:
prompt should be reset to '' when the user click on submit

<img width="779" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/867e3911-3efb-4221-88cf-19e1cd16a11e">